charts/kong-collectorapi/kong-values.yaml

```diff
@@ -0,0 +1,51 @@
+image:
+  # repository: kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition
+  # tag: latest
+  repository: mashape/kong-enterprise
```

Comment:
This seems wrong.

Reply:
Just temporarily using internal master builds until I've made a release for collector and collector-plugin is in an enterprise build.